entities.csv

```diff
@@ -196,6 +197,7 @@ DNA_polymerase_alpha
 DNA_polymerase_delta
 DNM
 DRD
+DUB

```

Comment:
Since DUBs are a big family/category, perhaps the identifier should be longer form so it's easily distinguishable from a gene (in result lists for example). Maybe `Deubiquitinase`?

Reply:
That’s an excellent suggestion. I will also double check mesh xrefs

Reply:
Done in 5895475 and 8647591